plan-a.md

```diff
@@ -0,0 +1,17 @@
+### Initial Plan to save Earth
+
+This is a **cool** plan
+- [ ] Put the team together
+  - Wolverine
+  - Superman
+  - Flash
+  - Green Arrow
```

Comment:
This cannot happen @d34d-p00l 
There is no crossover between DC and Marvel...
need to rethink your team strategy

Reply:
Doh .. headslap ... I'll fix it